client/src/pages/admin/MergePeople.js

```diff
@@ -1,388 +1,883 @@
-                      {loser && loser.uuid && (
-                        <fieldset>
-                          {showPersonDetails(new Person(loser))}
-                        </fieldset>
-                      )}
-                    </Row>
-                  </Col>
-                  <Col md={6}>
-                    <Row>
-                      <Field
-                        name="winner"
-                        label="Winner"
-                        component={FieldHelper.SpecialField}
-                        onChange={value => {
-                          // validation will be done by setFieldValue
-                          setFieldTouched("winner", true, false) // onBlur doesn't work when selecting an option
-                          setFieldValue("winner", value)
-                        }}
-                        vertical
-                        widget={
-                          <AdvancedSingleSelect
-                            fieldName="winner"
-                            placeholder="Select the OTHER duplicate person"
-                            value={values.winner}
-                            overlayColumns={["Name"]}
-                            overlayRenderRow={PersonSimpleOverlayRow}
-                            filterDefs={peopleFilters}
-                            objectType={Person}
-                            valueKey="name"
-                            fields={personFields}
-                            addon={PEOPLE_ICON}
-                          />
-                        }
-                      />
-                    </Row>
-                    <Row>
-                      {winner && winner.uuid && (
-                        <fieldset>
-                          {showPersonDetails(new Person(winner))}
-                        </fieldset>
-                      )}
-                    </Row>
-                  </Col>
-                </Row>
-                <Row>
-                  <Col md={12}>
-                    {canCopyPosition(loser, winner) && (
-                      <Field
-                        name="copyPosition"
-                        component={FieldHelper.SpecialField}
-                        label={null}
-                        widget={
-                          <FormCheck
-                            type="checkbox"
-                            label={`Set position on winner to ${loser.position.name}`}
-                            checked={values.copyPosition}
-                          />
-                        }
+    <Container fluid>
+      <Row>
+        <Messages error={saveError} />
+        <h4>Merge People Tool</h4>
+      </Row>
+      <Row>
+        <Col md={4} id="left-merge-per-col">
+          <PersonColumn
+            mergeState={mergeState}
+            dispatchMergeActions={dispatchMergeActions}
+            align={ALIGN_OPTIONS.LEFT}
+            label="Person 1"
+          />
+        </Col>
+        <Col md={4} id="mid-merge-per-col">
```

Comment:
Please put a fieldset in the background to be consistent with the rest of the application.